setup.cfg

```diff
@@ -0,0 +1,48 @@
+[metadata]
+name = pysize
+url = https://github.com/bosswissam/pysize
+project_urls = 
+    CI: Travis = https://travis-ci.com/bosswissam/pysize
+    GitHub: repo = https://github.com/bosswissam/pysize
+    GitHub: issues = https://github.com/bosswissam/pysize/issues
+author = bosswissam
+author_email = bosswissam@gmail.com
+description = Use to quickly measure the size of your python objects.
+long_description = file: README.md
```

Comment:
@bosswissam is it okay if we convert README into rst?

@serhii73 if we'll decide to keep Markdown, it'll require one extra option to be set in metadata, explicitly declaring the correct MIME type of the description.